Sources/NetworkProtection/Settings/VPNSettings.swift

```diff
@@ -91,53 +91,89 @@ public final class VPNSettings {
 
     private(set) public lazy var changePublisher: AnyPublisher<Change, Never> = {
 
-        let connectOnLoginPublisher = connectOnLoginPublisher.map { connectOnLogin in
-            Change.setConnectOnLogin(connectOnLogin)
-        }.eraseToAnyPublisher()
-
-        let includeAllNetworksPublisher = includeAllNetworksPublisher.map { includeAllNetworks in
-            Change.setIncludeAllNetworks(includeAllNetworks)
-        }.eraseToAnyPublisher()
-
-        let enforceRoutesPublisher = enforceRoutesPublisher.map { enforceRoutes in
-            Change.setEnforceRoutes(enforceRoutes)
-        }.eraseToAnyPublisher()
-
-        let excludeLocalNetworksPublisher = excludeLocalNetworksPublisher.map { excludeLocalNetworks in
-            Change.setExcludeLocalNetworks(excludeLocalNetworks)
-        }.eraseToAnyPublisher()
-
-        let notifyStatusChangesPublisher = notifyStatusChangesPublisher.map { notifyStatusChanges in
-            Change.setNotifyStatusChanges(notifyStatusChanges)
-        }.eraseToAnyPublisher()
-
-        let registrationKeyValidityPublisher = registrationKeyValidityPublisher.map { validity in
-            Change.setRegistrationKeyValidity(validity)
-        }.eraseToAnyPublisher()
-
-        let serverChangePublisher = selectedServerPublisher.map { server in
-            Change.setSelectedServer(server)
-        }.eraseToAnyPublisher()
-
-        let locationChangePublisher = selectedLocationPublisher.map { location in
-            Change.setSelectedLocation(location)
-        }.eraseToAnyPublisher()
-
-        let environmentChangePublisher = selectedEnvironmentPublisher.map { environment in
-            Change.setSelectedEnvironment(environment)
-        }.eraseToAnyPublisher()
-
-        let showInMenuBarPublisher = showInMenuBarPublisher.map { showInMenuBar in
-            Change.setShowInMenuBar(showInMenuBar)
-        }.eraseToAnyPublisher()
-
-        let vpnFirstEnabledPublisher = vpnFirstEnabledPublisher.map { vpnFirstEnabled in
-            Change.setVPNFirstEnabled(vpnFirstEnabled)
-        }.eraseToAnyPublisher()
-
-        let disableRekeyingPublisher = disableRekeyingPublisher.map { disableRekeying in
-            Change.setDisableRekeying(disableRekeying)
-        }.eraseToAnyPublisher()
+        let connectOnLoginPublisher = connectOnLoginPublisher
+            .dropFirst()
+            .removeDuplicates()
```

Comment:
I've added this for all publishers.  We could do this in the merged publisher, but I figured doing it here involves less CPU time spent on something we don't need.